lib/sites/thinkgeek.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const siteUtils = require('../site-utils');
+const logger = require('../logger')();
+
+class ThinkgeekSite {
+  constructor(uri) {
+    if (!ThinkgeekSite.isSite(uri)) {
+      throw new Error('invalid uri for Thinkgeek: %s', uri);
+    }
+    this._uri = uri;
+  }
+
+  getURIForPageData() {
+    return this._uri;
+  }
+
+  isJSON() {
+    return false;
+  }
+
+  findPriceOnPage($) {
+    const priceString = siteUtils.findContentOnPage($, ['form#buy h3']);
+    if (!priceString) {
+      logger.error('price not found on Thinkgeek page, uri: %s', this._uri);
+      return -1;
+    }
+    return siteUtils.processPrice(priceString);
+  }
+
+  findCategoryOnPage() {
+    const category = siteUtils.categories.OTHER; // TODO find the category
+    // logger.log('category: %s', category);
```

Comment:
Can you please add back in this log statement (rather than commenting it out)?  Just for consistency.